stdlib/2/typing.pyi

```diff
@@ -221,6 +221,9 @@ class MutableMapping(Mapping[_KT, _VT], Generic[_KT, _VT]):
     def __delitem__(self, v: _KT) -> None: ...
 
     def clear(self) -> None: ...
+    @overload
+    def pop(self, k: _KT) -> _VT: ...
+    @overload
```

Comment:
Interestingly, the get() overloads have `# type: ignore` comments. Looks like they are not needed anymore since your PR passes tests?